tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -188,28 +188,58 @@ install_initrd_sev() {
 	install_initrd "initrd-sev" "sev"
 }
 
+#Install kernel component helper
+install_cached_kernel_tarball_component() {
+	local kernel_name=${1}
+	local job_name=${2}
+
+	install_cached_tarball_component \
+		"${kernel_name}" \
+		"${jenkins_url}/job/${job_name}/${cached_artifacts_path}" \
+		"${kernel_version}-${kernel_kata_config_version}" \
+		"$(get_kernel_image_name)" \
+		"${final_tarball_name}" \
+		"${final_tarball_path}" \
+		|| return 1
+	
+	if [[ "${kernel_name}" != "kernel-sev" ]]; then
+		return 0
+	fi
+
+	# SEV specific code path
+	install_cached_tarball_component \
+		"${kernel_name}" \
+		"${jenkins_url}/job/${job_name}/${cached_artifacts_path}" \
+		"${kernel_version}-${kernel_kata_config_version}" \
+		"$(get_kernel_image_name)" \
+		"kata-static-kernel-sev-modules.tar.xz" \
+		"${workdir}/kata-static-kernel-sev-modules.tar.xz" \
+		|| return 1
+
+	mkdir -p "${module_dir}"
+	tar xvf "${workdir}/kata-static-kernel-sev-modules.tar.xz" -C  "${module_dir}" && return 0
+
+	return 1
+}
+
 #Install kernel asset
 install_kernel_helper() {
 	local kernel_version_yaml_path="${1}"
 	local kernel_name="${2}"
 	local extra_cmd=${3}
 
 	export kernel_version="$(get_from_kata_deps ${kernel_version_yaml_path})"
-	local kernel_kata_config_version="$(cat ${repo_root_dir}/tools/packaging/kernel/kata_config_version)"
+	export kernel_kata_config_version="$(cat ${repo_root_dir}/tools/packaging/kernel/kata_config_version)"
 	local job_name="kata-containers-main-${kernel_name}-$(uname -m)"
+	local module_dir=""
 
 	if [[ "${kernel_name}" == "kernel-sev" ]]; then
-		job_name="kata-containers-main-${kernel_name}-sev-$(uname -m)"
+		kernel_version="$(yq r $versions_yaml assets.kernel.sev.version)"
+		default_patches_dir="${repo_root_dir}/tools/packaging/kernel/patches"
+		module_dir="${repo_root_dir}/tools/packaging/kata-deploy/local-build/build/cc-sev-kernel/builddir/kata-linux-${kernel_version#v}-${kernel_kata_config_version}/lib/modules/${kernel_version#v}"
```

Comment:
@UnmeshDeodhar one more nit...sorry...

The kernel modules buildroot is build/cc-sev-kernel/builddir, whereas the kernel-sev buildroot is build/kernel-sev/builddir. Could you change this line to build the modules under build/kernel-sev/builddir as well?

Reply:
Changed.